mappings/net/minecraft/world/biome/Biome.mapping

```diff
@@ -13,13 +15,19 @@ CLASS none/ajj net/minecraft/world/biome/Biome
 		METHOD a getName (Lnone/ajj$a;)Ljava/lang/String;
 		METHOD b setRainfall (F)Lnone/ajj$a;
 			ARG 0 rainfall
+		METHOD c setHeight (F)Lnone/ajj$a;
+			ARG 0 value
+		METHOD d setScale (F)Lnone/ajj$a;
+			ARG 0 scale
```

Comment:
`value`
Also I'm seeing `setRainfall` doesn't use `value` either. Fix that too pls :smile: